app/assets/stylesheets/new_styles/_registration.scss

```diff
@@ -49,3 +49,34 @@
     }
   }
 }
+
+#welcome-to-diaspora {
+  -webkit-box-shadow: inset 0 -2px 10px rgba(0,0,0,0.35);
+  -moz-box-shadow: inset 0 -2px 10px rgba(0,0,0,0.35);
+  position: relative;
+  width: 100%;
+  left: 0;
+  top: 0;
+  padding: 10px;
+  .right {
+    position: absolute;
+    right: 0;
+    top: 10px;
+  }
+  h1,h3 {
+    color: #fff;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  background: orange;
+  &:hover {
+    #gs-skip-x {
+      @include opacity(0.4);
```

Comment:
opacity is supported unprefixed these days, 
`opacity: 0.4;` would be better. See #5532